services/data_plane_manager/src/test/java/com/futurewei/alcor/dataplane/utils/GoalStateManagerTest.java

```diff
@@ -4540,703 +4540,11 @@ public void scenarioB() {
     check(input, output);
   }
 
-  @Test
```

Comment:
You don't have to remove the test case, temporarily commenting out "@Test" will make it :)

Let us get back the test cases when the new change is verified locally.

Reply:
fixed in 0fc306c